src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -1466,6 +1467,90 @@ namespace OpenLoco::Ui::WindowManager
         invalidateWidget(window->type, window->number, widgetIndex);
     }
 
+    static bool isStepperGroup(Window& w, WidgetIndex_t index, WidgetType buttonType)
+    {
+        const auto& widgets = w.widgets;
+
+        if (widgets[index].type != WidgetType::textbox && widgets[index].type != WidgetType::wt_3)
+            return false;
+
+        if (widgets[index + 1].type != buttonType)
+            return false;
+
+        if (widgets[index + 2].type != buttonType)
+            return false;
+
+        return true;
+    }
+
+    static std::optional<WidgetIndex_t> getStepperGroupWidgetIndex(Window& w, WidgetIndex_t startIndex)
+    {
+        // We only iterate 3 times as we might be at the textbox or one of the buttons.
+        for (WidgetIndex_t index = 0; index < 3; index++)
+        {
+            const auto reverseIndex = startIndex - index;
+            if (reverseIndex < 0)
+            {
+                break;
+            }
+
+            if (isStepperGroup(w, reverseIndex, WidgetType::toolbarTab))
```

Comment:
I realise this has been in the code for a while but @AaronVanGeffen any idea why we are checking toolbarTab here? To me it doesn't make any sense you. ToolbarTabs are only for the toolbar so its never going to be a stepper. Perhaps this was meant to be a tab rather than a ToolbarTab.

Reply:
I think you may be right. In OpenRCT2, `WindowWidgetType::TrnBtn` is used, which is the equivalent of this. It likely suffers the same issue. Maybe @Broxzier remembers?

Reply:
Pretty sure I just copied the condition from somewhere else.